docs/website/binding_types_reference_guide.md

```diff
@@ -380,6 +380,50 @@ that contains the `MakeBackgroundRed` extension method.   This means
 that you can now call "MakeBackgroundRed" on any `UIView` subclass,
 giving you the same functionality you would get on Objective-C.
 
+In some cases you will find **static** members inside categories like in the following example:
+
+```objc
+@interface FooObject (MyFooObjectExtension)
++ (BOOL)boolMethod:(NSRange *)range;
+@end
+```
+
+This will lead to an **incorrect** Category C# interface definition:
+
+```csharp
+[Category]
+[BaseType (typeof (FooObject))]
+interface FooObject_Extensions {
+
+	// Incorrect Interface definition
+	[Static]
+	[Export ("boolMethod:")]
+	bool BoolMethod (NSRange range);
+}
+```
+
+This is incorrect because in order to use the `BoolMethod` extension you need an instance of `FooObject` but you are binding an ObjC **static** extension, this is a side effect due to the fact of how C# extension methods are implemented. 
+
+The only way to use the above definitions is by the following ugly code:
+
+```csharp
+(null as FooObject).BoolMethod (range);
+```
+
+The solution to this problem is simple, just inline the `BoolMethod` definition inside the `FooObject` interface definition itself, this will allow you to call this extenion like it is intended `FooObject.BoolMethod (range)`.
```

Comment:
It's more *one* solution (maybe the easiest), from a few ones, than **the** solution.